tests/test_edx_east_roles.sh

```diff
@@ -38,6 +38,10 @@ ansible-playbook -i localhost, -c local --tags edxapp_cfg edxapp.yml -e edxapp_u
     -e@languages.yml \
     -e@server-vars.yml \
     -e EDXAPP_PREVIEW_SITE_NAME=""
```

Comment:
you need trailing \ (back slashes) before the newline so bash knows this is all one command.